cloud.foundry.cli/build.gradle

```diff
@@ -60,8 +62,8 @@ dependencies {
 
 test {
     useJUnitPlatform()
-
     maxHeapSize = '1G'
+    finalizedBy jacocoTestReport // report is always generated after tests run
```

Comment:
See above. Does it make sense to always run the command, even though e.g., during development, it isn't really needed? I think it would make more sense to just run this on the CI. However, I don't know how big the impact of this line is on the test runtime, i.e., how much longer the `test` command runs when this is activated. If it's just a second or two, then let's just keep it as-is.

Reply:
Agree with the optinion of @M4eqP

Reply:
I don't think this is a big deal. But I guess we can also configure it in the way that it is only running on the CI somehow.